config/src/main/java/org/mobilitydata/gtfsvalidator/config/DefaultConfig.java

```diff
@@ -51,6 +53,7 @@
     private final RawFileRepository rawFileRepo = new InMemoryRawFileRepository();
     private final ValidationResultRepository resultRepo = new InMemoryValidationResultRepository();
     private final GtfsDataRepository gtfsDataRepository = new InMemoryGtfsDataRepository();
+    private final TimeUtils timeUtils = new TimeConversionUtils();
```

Comment:
`TimeConversionUtils` shall implement a singleton pattern. https://www.geeksforgeeks.org/singleton-class-java/ 
We'd retrieve and pass the single instance here